04-behavr.Rmd

```diff
@@ -175,9 +175,7 @@ dt[meta = TRUE]
 
 `paste()` is a function that link strings of characters with an arbitrary separator (`"|"` here).
 
-We can also add new metavariables from a summary, in this example we compute the average time moving per fly and then we add it to the metadata.
-
-
+We can also add new metavariables from a summary, see [Summarise data][Summarise data]
```

Comment:
If you are trying to link a section, see the second answer here:
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown.
so `[Summarise data](#summarise-data).`. Then no need for the `[...]` in `Summarise data{-}[Summarise data]`. :D